x-pack/plugins/apm/public/components/shared/links/apm/transaction_detail_link.tsx

```diff
@@ -68,5 +77,32 @@ export function TransactionDetailLink({
     search: location.search,
   });
 
-  return <EuiLink href={href} {...rest} />;
+  return (
+    <>
+      {transactionName === txGroupsDroppedBucketName ? (
+        <EuiToolTip
+          content={i18n.translate(
+            'xpack.apm.transactionDetail.tooltip.message',
+            {
+              defaultMessage:
+                "The transaction group limit has been reached. Please see the APM Server docs for 'aggregation.transaction.max_groups' to increase this",
+            }
+          )}
+        >
+          <EuiFlexGroup alignItems="center" gutterSize="xs">
+            <EuiFlexItem grow={1}>
+              {i18n.translate('xpack.apm.transactionDetail.other.label', {
+                defaultMessage: 'other',
+              })}
+            </EuiFlexItem>
+            <EuiFlexItem grow={1}>
+              <EuiIcon size="s" color="subdued" type="alert" />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiToolTip>
+      ) : (
+        <EuiLink href={href} {...rest} />
+      )}
+    </>
+  );
```

Comment:
Long jsx ternaries can be difficult to read. In this case an if-block is simpler:

```suggestion
  if (transactionName === txGroupsDroppedBucketName) {
    return <EuiLink href={href} {...rest} />;
  }

  return (
    <EuiToolTip
      content={i18n.translate("xpack.apm.transactionDetail.tooltip.message", {
        defaultMessage:
          "The transaction group limit has been reached. Please see the APM Server docs for 'aggregation.transaction.max_groups' to increase this",
      })}
    >
      <EuiFlexGroup alignItems="center" gutterSize="xs">
        <EuiFlexItem grow={1}>
          {i18n.translate("xpack.apm.transactionDetail.other.label", {
            defaultMessage: "other",
          })}
        </EuiFlexItem>
        <EuiFlexItem grow={1}>
          <EuiIcon size="s" color="subdued" type="alert" />
        </EuiFlexItem>
      </EuiFlexGroup>
    </EuiToolTip>
  );

```